webext-base.css

```diff
@@ -84,6 +83,24 @@ kbd {
 	}
 }
 
+/* Safari-specific style */
+@supports (-webkit-text-security: none) {
+	:root {
+		/* Textfields on Safari desktop are too thin, they expect a darker background */
+		background-color: Window;
+	}
+}
```

Comment:
This gray background looks good when coupled with the small `font:-apple-system-body;` text and a small windows, but not on full-width tabs, especially if they have extra style like for Refined GitHub.

Dropping it…